tmt/cli.py

```diff
@@ -217,6 +238,9 @@ def provision(context, **kwargs):
 @click.option(
     '-h', '--how', metavar='METHOD',
     help='Use specified method for environment preparation.')
+@click.option(
+    '-s', '--script', metavar='SCRIPT',
+    help='Script, inline script, or path to ansible playbook to execute.')
```

Comment:
For ansible it would be better to also have --playbook option

Reply:
I'll make the changes first thing tomorrow (unf. I'm sick).
Ideally, the name for this should be not specific, but a general one: like 'config' | 'script' | 'file' | 'source' file. Can you think of any name that's fitting?
WRT `fmf` config, I've created aliases to account for that, but then, but if 'there shlould be only one and obious way', we should limit that-  please check those (the `self.alias(...)` calls).

Reply:
@psss FYI ^

Reply:
> Thanks for the review! I'll make the fixes in the morning.

Sorry for taking so long to do it :(

Reply:
I've renamed it to `input`, but I've kept all the aliases. I think we can provide multiple aliases for click also if needed.